docs/reference/custom_fields/fields_list.md

```diff
@@ -0,0 +1,98 @@
+# Fields list
+
+_Introduced in InvenioRDM v12_
+
+The following document is a reference guide for all the custom fields implemented and distributed in invenio.
+
+!!!tip "How to add a field to an instance?"
+    If you want to add a field to an instance, please refer to the custom fields [documentation](../../customize/metadata/custom_fields/records.md#add-reusable-custom-fields-to-records).
+
+## Publishing information
+
+This group of fields contains the metadata for "Journal", "Imprint" and "Thesis". Each field has its own data representation, however they are grouped in the UI.
+
+**Deposit form** 
+
+![Deposit form](../../images/pub_info_deposit_form.png)
+
+**Landing page**
+
+![Landing page](../../images/pub_info_landing_page.png)
+
+Publishing information is displayed in the section "Details", under "Published in".
+
+### Journal
+
+This nested field implements a journal's metadata. It can be used to describe a journal where an article was published.
+
+**Metadata**
+
+- **title** `String`: The title of the journal.
+- **volume** `String`: The volume where the article was published.
+- **issue** `String`: The issue within the volume.
+- **pages** `String`: The pages within the issue where the article was published. It can be a number or a range in any format.
+
+### Imprint
+
+This nested field implements an imprint's metadata. It can be used to describe a book, report or chapter where a record was published.
+
+**Metadata**
+
+- **title** `String`: The title of the book or report where the record was published.
+- **isbn** `String`: The book's International Standard Book Number. Applies if the imprint is a book.
+- **place** `String`: Location where the book (or report) was published.
+- **pages** `String`: The pages within the book or report. It can be a number or a range in any format.
+
+### Thesis
+
+This field implements thesis metadata, more specifically an awarding university. A thesis supervisor is inferred from the list of contributors whose role is "Supervisor".
```

Comment:
It is not in the UI. What I mean was that the supervisor was removed from this field since it was redundant, it can be found in the contributors list.

Thanks!

Reply:
minor: could be `added as a contributor with _supervisor_ role`? The whole documentation talks about input data, and saying `found` leaves the quesiton open to "would it be autocompleted? or how/why would it appear there?"

Reply:
The idea here is that we "removed" the attribute "supervisor" from this field. 

And we did it because it can be found already in the contributors list. I can see this wordig being relevant if someone is implementing a serializer.